javascript/modules/common/src/errors.ts

```diff
@@ -43,3 +43,9 @@ export class UnexpectedActionError extends Error {
     super("The received action is different from the expected one");
   }
 }
+
```

Comment:
Did NoSignerError() already exist?

Reply:
Yes, I just added an error for when the token address is not specified when trying to update the allowance because you need the token address